pkg/odo/cli/registry/util/util.go

```diff
@@ -0,0 +1,32 @@
+package util
+
+import (
+	// odo packages
+
+	"os"
+
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/preference"
+	"github.com/pkg/errors"
+)
+
+// IsSecure checks if the registry is secure
+func IsSecure(registryName string) bool {
```

Comment:
Can you add unit tests for this function if you can mock the preferences file? I think there's some other unit tests in odo that do the same

Reply:
Yep, I can add that.